magda-typescript-common/src/CkanClient.ts

```diff
@@ -0,0 +1,364 @@
+import fetch from "isomorphic-fetch";
```

Comment:
Do we need this for main repo?
It's only used by the minion

Reply:
Oops. Yeah we don't need this file in the main repo. I thought I would delete it after the minion works, and then promptly forgot about it :smile: